automatic/displayfusion/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,19 @@
+﻿$ErrorActionPreference  = 'Stop'
+
+$downloadDir        = Get-PackageCacheLocation
+$installer          = 'DisplayFusionSetup-8.1.2.exe'
+$programFiles       = Join-Path ${env:ProgramFiles(x86)} 'DisplayFusion'
+$localeTwoLetter    = (Get-Culture).TwoLetterISOLanguageName
+$arguments          = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = 'DisplayFusion'
+    file            = Join-Path $downloadDir $installer
```

Comment:
remove this property, this will cause the automated tests on chocolatey to fail.